packages/block-library/src/table/style.scss

```diff
@@ -1,6 +1,7 @@
 .wp-block-table {
 	// Fixed layout toggle
 	&.has-fixed-layout tbody {
+		display: table;
 		table-layout: fixed;
```

Comment:
Why is this not used on the `table` element? This seems a bit odd.

Reply:
Same reason as noted in https://github.com/WordPress/gutenberg/pull/9306#discussion_r213971788 essentially.